dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JsonConfigTest.java

```diff
@@ -78,119 +75,119 @@ public void testError() throws Exception {
+        line = checkMessageLogUpdate(true, ALL_SOURCE_LIST, "");
         //console
         RemoteFile consoleLogFile = server.getConsoleLogFile();
-        consoleline = checkConsoleLogUpdate(false, consoleLogFile, "INFO", sourceList, "");
+        consoleline = checkConsoleLogUpdate(false, consoleLogFile, "INFO", ALL_SOURCE_LIST, "");
     }
 
     //<logging consoleLogLevel="INFO" consoleFormat="json" messageFormat="json"/>
     @Test
     public void testJsonConsoleJsonMessage() throws Exception {
         line = setConfig(SERVER_XML_JSON_CONSOLE_JSON_MESSAGE);
         Assert.assertNotNull("Both CWWKG0017I and CWWKG0018I are not found", line);
-        line = checkMessageLogUpdate(true, sourceList, "");
+        line = checkMessageLogUpdate(true, ALL_SOURCE_LIST, "");
         RemoteFile consoleLogFile = server.getConsoleLogFile();
-        consoleline = checkConsoleLogUpdate(true, consoleLogFile, "INFO", sourceList, "");
+        consoleline = checkConsoleLogUpdate(true, consoleLogFile, "INFO", ALL_SOURCE_LIST, "");
     }
 
     //<logging consoleLogLevel="INFO" consoleFormat="basic" messageFormat="basic" maxFileSize="100"/>
     @Test
     public void testBasicConsoleBasicMessage() throws Exception {
         line = setConfig(SERVER_XML_BASIC_CONSOLE_BASIC_MESSAGE);
         Assert.assertNotNull("Both CWWKG0017I and CWWKG0018I are not found", line);
-        line = checkMessageLogUpdate(false, sourceList, "");
+        line = checkMessageLogUpdate(false, ALL_SOURCE_LIST, "");
         RemoteFile consoleLogFile = server.getConsoleLogFile();
-        consoleline = checkConsoleLogUpdate(false, consoleLogFile, "INFO", sourceList, "");
+        consoleline = checkConsoleLogUpdate(false, consoleLogFile, "INFO", ALL_SOURCE_LIST, "");
     }
 
     //<logging consoleLogLevel="INFO" consoleFormat="basic" messageFormat="json" maxFileSize="100"/>
     @Test
     public void testBasicConsoleJsonMessageMaxFileSize() throws Exception {
         line = setConfig(SERVER_XML_BASIC_CONSOLE_JSON_MESSAGE_MAXFILESIZE);
         Assert.assertNotNull("Both CWWKG0017I and CWWKG0018I are not found", line);
-        line = checkMessageLogUpdate(true, sourceList, "");
+        line = checkMessageLogUpdate(true, ALL_SOURCE_LIST, "");
         RemoteFile consoleLogFile = server.getConsoleLogFile();
-        consoleline = checkConsoleLogUpdate(false, consoleLogFile, "INFO", sourceList, "");
+        consoleline = checkConsoleLogUpdate(false, consoleLogFile, "INFO", ALL_SOURCE_LIST, "");
     }
 
     //<logging consoleLogLevel="INFO" consoleFormat="basic" messageFormat="json" messageSource="trace,accesslog" maxFileSize="100"/>
     @Test
     public void testJsonMessageSrcTraceAccess() throws Exception {
-        ArrayList<String> messagesourceList = new ArrayList<String>(Arrays.asList("trace", "accesslog"));
+        ArrayList<String> messageALL_SOURCE_LIST = new ArrayList<String>(Arrays.asList("trace", "accesslog"));
```

Comment:
in previous review comments, I just meant to change the one static variable to ALL_SOURCE_LIST.  In places where it's not a static variable, should just keep the old names.  eg. messageSourceList here.